docs/6.x/pack.md

```diff
@@ -733,9 +733,8 @@ To see more examples of specific hooks, please refer to the following documentat
 
 ## Helm Integration
 
-It is possible to use [Helm](https://docs.helm.sh/) charts as a way to package
-and install applications as every Gravity Cluster comes with a preconfigured
-Tiller server and its client, Helm.
```

Comment:
Have we removed Tiller because we are on Helm 3.0 . If so, I would call that our specifically.

Reply:
No, not yet. We haven't upgraded to Helm 3 yet.

Reply:
But I figured this is implementation detail and not really necessary here and also this way we won't have to change it when we actually do upgrade.

Reply:
I would suggest we add a  

```
!!! note
 Gravity 6.0 works with Helm 2.0. We are currently working on Helm 3.0 Integration.

Reply:
Yeah, that would be good to have. I'll add this note.